docs/admin/cluster-management.md

```diff
@@ -136,7 +136,7 @@ The initial values of the autoscaler parameters set by ``kube-up.sh`` and some m
 or using gcloud CLI:
 
 ```
-gcloud preview autoscaler --zone compute-zone <command>
+gcloud alpha compute autoscaler --zone $ZONE <command>
```

Comment:
The implication here is that the user has to specify which zone they want.  I think `$ZONE` is more obviously a variable than `compute-zone`